tscached/kquery.py

```diff
@@ -22,6 +24,16 @@ def from_request(cls, request, redis_client):
         for metric in request.get('metrics', []):
             new = cls(redis_client)
             new.query = metric
+
+            # This seemingly terrifying kludge makes it easy to merge chunks of MTS together.
+            # For all aggregators, we remove align_sampling and add align_start_time.
+            # Otherwise, partial windows (if aggregated) will appear on every seam in the data. Info:
+            # https://kairosdb.github.io/docs/build/html/restapi/QueryMetrics.html#metric-properties
+            for agg_ndx in xrange(len(new.query.get('aggregators', []))):
+                if 'align_sampling' in new.query['aggregators'][agg_ndx]:
+                    del new.query['aggregators'][agg_ndx]['align_sampling']
```

Comment:
Just following the KairosDB spec - they claim that setting both values leads to unpredictable behavior. Same difference, I figure.